acceptance-tests/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
```

Comment:
is it normal to have this change in this PR ?

Reply:
I added it since it was missing and we had no logs, and wanted to avoid creating a dedicated PR only for this